pkg/sql/sem/builtins/replication_builtins.go

```diff
@@ -300,6 +300,7 @@ var replicationBuiltins = map[string]builtinDefinition{
 		tree.Overload{
 			Types: tree.ArgTypes{
 				{"stream_id", types.Int},
+				{"ingestion_cutover", types.Bool},
```

Comment:
Do we ever not want to do this? I.e. do we imagine this function, which intended to be invoked from destination cluster, will ever want to invoke this set to false?

Is the intention to store some information about cutover event in the src cluster? If so, perhaps having a timestamp makes more sense.  Or even some sort of proto message indicating the reason for cutover, timestamp, etc..?